packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -131,6 +132,9 @@ export class InputManager {
      * @returns Mesh that the pointer is pointer is hovering over
      */
     public get meshUnderPointer(): Nullable<AbstractMesh> {
+        if (this._movePointerInfo && this._movePointerInfo.pickInfo) {
+            this._movePointerInfo = null;
+        }

```

Comment:
You only want to call `pickInfo` once right?
```suggestion
        if (this._movePointerInfo) {
            // Calling pickInfo will populate `this._pointerOverMesh` indirectly.
            this._movePointerInfo.pickInfo;
            this._movePointerInfo = null;
        }
```

Reply:
Discussed offline